examples/basics/quick_start.ipynb

```diff
@@ -0,0 +1,195 @@
+{
```

Comment:
<p>Suggestion: Use <code>export_v2()</code> instead which is easier to grasp.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>